ServerNotifications.Web/Domain/Notifier.cs

```diff
@@ -1,26 +1,40 @@
 ﻿using ServerNotifications.Web.Domain.Twilio;
 using ServerNotifications.Web.Models.Repository;
+using System;
+using System.Collections.Generic;
+using Twilio.Clients;
 using WebGrease.Css.Extensions;
 
 namespace ServerNotifications.Web.Domain
 {
     public class Notifier
     {
-        private const string ImageUrl = "http://howtodocs.s3.amazonaws.com/new-relic-monitor.png";
+        private Uri ImageUrl = new Uri("http://howtodocs.s3.amazonaws.com/new-relic-monitor.png");
```

Comment:
The naming applied to this one is use an underscore to prefix a private field. `_imageUrl`. The previous version uses _CamelCase_ because it was a constant.